### ISSUE TITLE
install fails if tables are already present

### ISSUE THREAD

**Issue by olets** [verbb/formie#655]
**Description**

After partially reverting a database, I was in a state where the dependency was installed by Composer, Formie was showing as disabled in the Craft CP, and "installing" (activating) via the CP failed. Attempting to install via the command line revealed the problem: the installation script doesn't handle the possibility that the tables are already present.

In this case it was my fault for (somehow…) getting my db into a state where Formie wasn't active, but it'd be nice if Formie didn't error.

**Steps to reproduce**
1. I'm not sure. Sorry!

### Form settings
Please pick from the options below.

**Additional info**
- Plugin version: 1.4.25
- Craft version: 3.7.21
- Multi-site: No

**Additional context**

First attempt:

```shell
% ./craft plugin/install formie
*** installing formie
    > create table {{%formie_emailtemplates}} ...Exception: SQLSTATE[42S01]: Base table or view already exists: 1050 Table 'formie_emailtemplates' already exists
The SQL being executed was: CREATE TABLE `formie_emailtemplates` (
        `id` int(11) NOT NULL AUTO_INCREMENT PRIMARY KEY,
        `name` varchar(255) NOT NULL,
        `handle` varchar(64) NOT NULL,
        `template` varchar(255) NOT NULL,
        `sortOrder` smallint(6) UNSIGNED,
        `dateDeleted` datetime(0),
        `dateCreated` datetime(0) NOT NULL,
        `dateUpdated` datetime(0) NOT NULL,
        `uid` char(36) NOT NULL DEFAULT '0'
) ENGINE = InnoDb DEFAULT CHARACTER SET = utf8 DEFAULT COLLATE = utf8_unicode_ci (/var/www/html/vendor/yiisoft/yii2/db/Schema.php:678)
```

Deleted all tables with the `formie_` prefix (if I'd written them down maybe we'd have a clue about the steps to reproduce, but I didn't). Second attempt:

```shell
% docker-compose exec php ./craft plugin/install formie
*** installing formie
# snip
*** installed formie successfully (time: 2.355s)
```

**Comment by engram-design** [comment]
I'll consider this, but we don't do this in any of our other plugins, because Craft (and their own plugins) don't do this.

The tricky part is we can certainly do a check "if table exists, skip" the problem is we can't know for sure the schema of the table doesn't match what we expect. I believe that's why its a all-or-nothing approach, so that the table schema is always "fresh".

I'll ponder further about it!

**Comment by olets** [comment]
Good to know, I guess the state I got my db into is more unusual than I thought — I haven't hit an error like this before, and imagined that meant Formie was missing some usual check. I'll leave this open because it sounds like you're interested in the idea, but feel free to close.

**Comment by engram-design** [comment]
Yep, I might query the P&T about it for some best-practice guidance.

**Comment by engram-design** [comment]
Addressed in [2.0.0](https://github.com/verbb/formie/releases/tag/2.0.0)
